src/main/java/com/openhack/toyland/service/DropDownBoxService.java

```diff
@@ -0,0 +1,46 @@
+package com.openhack.toyland.service;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import com.openhack.toyland.domain.OrganizationRepository;
+import com.openhack.toyland.domain.skill.SkillRepository;
+import com.openhack.toyland.domain.toy.Category;
+import com.openhack.toyland.domain.toy.Period;
+import com.openhack.toyland.dto.OrganizationResponse;
+import com.openhack.toyland.dto.SkillResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DropDownBoxService {
+
+	private SkillRepository skillRepository;
+	private OrganizationRepository organizationRepository;
+
+	@Autowired
+	DropDownBoxService(SkillRepository skillRepository, OrganizationRepository organizationRepository) {
+		this.skillRepository = skillRepository;
+		this.organizationRepository = organizationRepository;
+	}
+
```

Comment:
```suggestion
@RequiredArgsConstructor
@Service
public class DropDownBoxService {

	private SkillRepository skillRepository;
	private OrganizationRepository organizationRepository;

```

이렇게 수정하기로 소연님과 이야기가 되었습니다!

Reply:
수정했습니다~~
의존성 주입 방법이 3가지가 있네용~~ https://landsnail.tistory.com/entry/Spring-%EC%9D%98%EC%A1%B4%EC%84%B1-%EC%A3%BC%EC%9E%85-%EB%B0%A9%EB%B2%95

Reply:
덕분에 배워갑니다!